src/python/dxpy/nextflow/nextflow_templates.py

```diff
@@ -66,11 +66,10 @@ def get_nextflow_src(custom_inputs=None, profile=None, resources_dir=None):
     for i in custom_inputs:
         value = "${%s}" % (i['name'])
         if i.get("class") == "file":
-            value = "dx://$(jq .[$dnanexus_link] -r <<< ${%s})" % i['name']
-
+            value = "dx://${DX_WORKSPACE_ID}:/$(echo ${%s} | jq .[$dnanexus_link] -r | xargs -I {} dx describe {} --json | jq -r .name)" % i['name']
```

Comment:
Would it be possible to break it down to make it more readable (even create a separate function for this)

Reply:
This can be done under a separate ticket but IMHO it's important for debugging and readability later.